pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -47,12 +47,7 @@ type probeRes struct {
 var iteration uint32
 var serverNameAndPort string
 var addrChgTime time.Time // record last time intf address change time
-var zcloudCtx = zedcloud.ZedCloudContext{
-	FailureFunc:        zedcloud.ZedCloudFailure,
-	SuccessFunc:        zedcloud.ZedCloudSuccess,
-	TlsConfig:          &tls.Config{InsecureSkipVerify: true},
```

Comment:
Why not set tlsconfig here as well?

Reply:
I could not figure out how to do this in var definition here.